ui/packages/catalog-ui-search/src/main/webapp/component/query-add/query-add.view.js

```diff
@@ -54,6 +54,11 @@ module.exports = Marionette.LayoutView.extend({
     this.listenTo(user.getQuerySettings(), 'change:template', querySettings =>
       this.updateCurrentQuery(querySettings)
     )
+    this.listenTo(
```

Comment:
I think it would be simpler if we moved this to query-setting's initialize method. then we could set up the listener like 
```
this.listenTo(user.getQuerySettings(),
      'change:defaultResultFormId',
     () => this.renderResultForms(this.resultFormCollection.filteredList)
    )
```
and we won't need to set the default id to the model, we could access it directly from the user QuerySettings in `renderResultForms` method

Reply:
I don't think I have reference to `user` inside query-settings

Reply:
`user` is a singleton instance that can be imported in query-settings just as it is here https://github.com/codice/ddf/pull/5957/files#diff-5f1ab184f26124726515c4d001db1faaR28